From 9bb382493bb3cd39b06233d4c1d087be396a884a Mon Sep 17 00:00:00 2001
From: Kevin Cernekee <cernekee@gmail.com>
Date: Thu, 14 Apr 2016 22:26:07 -0700
Subject: [PATCH 3/8] NaCl: Disable IPV6_PATHMTU getsockopt() call

Unfortunately this feature is not yet supported under NaCl, and it
results in a compile error.

Signed-off-by: Kevin Cernekee <cernekee@gmail.com>
---
 dtls.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dtls.c b/dtls.c
index 3b9cbbe..a875255 100644
--- a/dtls.c
+++ b/dtls.c
@@ -1265,7 +1265,7 @@ static int detect_mtu_ipv6(struct openconnect_info *vpninfo, unsigned char *buf)
 		break;
 	} while(max_resends-- > 0);
 
-#ifndef _WIN32
+#if !defined(_WIN32) && !defined(__native_client__)
 	/* If we received back our DPD packet, do nothing; otherwise,
 	 * attempt to get MTU from the ICMP6 packet we received */
 	if (ret <= 0) {
-- 
1.9.1

